src/components/ManaCalculator/hooks/useManaState.ts

```diff
@@ -153,6 +153,15 @@ export function useGivenManaState(
     setState({
       ...state,
       userType: value,
+      [getStakedOrDelegated(value)]: state.heldTokens,
+      ...(value === UserType.VALIDATOR
+        ? {
+            validator: {
+              ...state.validator,
+              attractedNewDelegatedStake: state.heldTokens,
```

Comment:
`attractedNewDelegatedStake` is not `heldTokens`, there is a formula https://github.com/iotaledger/iota-wiki/blob/main/src/components/ManaCalculator/hooks/useManaState.ts#L235, it doesnt depend on held tokens but it depends on stakedTokens and the validators set